### PR TITLE
Create archivenode.bash

### PR DESCRIPTION
**Pull request by faddat**
make an archive node

### CODE REVIEW

File: scripts/archivenode.bash [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+git checkout v1.0.1
```
Comment (ValarDragon) [inline scripts/archivenode.bash:2]:
```suggestion
git checkout v3.2.0
```

No reason to ever be using anything older, since its all state compatible!

**Review by ValarDragon — APPROVED** [review]

**Comment by ValarDragon** [comment]
We should evolve this in the future to setup the cosmovisor folder fully, but this looks great to me! Thanks for making it

File: scripts/archivenode.bash (hunk 1 above)
Comment (faddat) [reply scripts/archivenode.bash:2]: are you certain of this?

I've found it to explode unless following this pattern

**Comment by faddat** [comment]
any time.  Also I now see what you mean about the docker.  I will "fix" it.

(remove arm build)

**Comment by ValarDragon** [comment]
oh huh, not sure why I didn't merge this earlier

**Pull request merged**